pkg/api/v1/types.go

```diff
@@ -2229,6 +2229,11 @@ type DeleteOptions struct {
 	// specified type will be used.
 	// Defaults to a per object value if not specified. zero means delete immediately.
 	GracePeriodSeconds *int64 `json:"gracePeriodSeconds"`
+
+	// If true, when deleting the pod managed by a replication controller, scheduler will try to put the replacement
```

Comment:
Slightly different wording that might be a bit clearer: "If the object to be deleted is a pod managed by a controller, setting to true requests that the replacement pod that the controller creates (if the controller decides to create such a pod) be scheduled to a different node than the one where it is currently running. Otherwise has no effect."